Refresh.GameServer/Database/GameDatabaseContext.Users.cs

```diff
@@ -100,12 +100,38 @@ public void UpdateUserData(GameUser user, SerializedUpdateData data, TokenGame g
 
             // ReSharper disable once InvertIf
             if (data.IconHash != null)
-                //PSP icons are special and use a GUID system separate from the mainline games,
-                //so we separate PSP icons to another field
-                if (game == TokenGame.LittleBigPlanetPSP)
-                    user.PspIconHash = data.IconHash;
-                else
-                    user.IconHash = data.IconHash;
+                switch (game)
+                {
+
+                    case TokenGame.LittleBigPlanet1:
+                    case TokenGame.LittleBigPlanet2:
+                    case TokenGame.LittleBigPlanet3:
+#if false //Enable this code once https://github.com/LittleBigRefresh/Refresh/issues/309 is resolved
+                        //If the icon is a remote asset, then it will work on Vita as well, so set the Vita hash 
+                        if (!data.IconHash.StartsWith('g'))
+                        {
+                            user.VitaIconHash = data.IconHash;
+                        }
+#endif
+                        user.IconHash = data.IconHash;
+                        break;
+                    case TokenGame.LittleBigPlanetVita:
+#if false //Enable this code once https://github.com/LittleBigRefresh/Refresh/issues/309 is resolved
```

Comment:
```suggestion
#if false // TODO: Enable this code once https://github.com/LittleBigRefresh/Refresh/issues/309 is resolved
                        //If the icon is a remote asset, then it will work on Vita as well, so set the Vita hash 
                        if (!data.IconHash.StartsWith('g'))
                        {
                            user.VitaIconHash = data.IconHash;
                        }
#endif
                        user.IconHash = data.IconHash;
                        break;
                    case TokenGame.LittleBigPlanetVita:
#if false // TODO: Enable this code once https://github.com/LittleBigRefresh/Refresh/issues/309 is resolved
```